role-desc.html

```diff
@@ -0,0 +1,612 @@
+
+      <!--SideBar-->
+      <div class="sidebar">
+        <div>
+          <div class="company-logo-area">
+            <div class="sidebar-brand">
+              <div class="tezo-logo">
+                <img src="./assets/TezoLogo.svg" alt="Tezo Logo" />
+              </div>
+            </div>
+          </div>
+          <div class="sidebar-list">
+            <div class="all-section">
+              <div class="underline-heading-container">
+                <div class="underline"></div>
+                <div class="sidebar-list-heading">ALL</div>
+              </div>
+              <div class="sub-section">
+                <div class="sub-sec dashboard item unlocked">
+                  <div class="sub-sec-left">
+                    <div class="sub-sec-left-left">
+                      <img
+                        src="./assets/dashboard.svg"
+                        alt=""
+                        class="sidebar-dashboard-icon" />
+                      <h4>Dashboard</h4>
+                    </div>
+                    <div class="expand-collapsed-icons">
+                      <span class="material-icons-outlined collapsed">
+                        chevron_right
+                      </span>
+                      <span class="material-icons-outlined expand">
+                        expand_more
+                      </span>
+                    </div>
+                  </div>
+                  <div class="sub-sec-right"></div>
+                </div>
+                <div
+                  class="sub-sec employees item unlocked"
+                  onclick="toggleSubSecClass(this, ['.employees-container','.alph-sort', '.function-container.employees',])">
+                  <div class="sub-sec-left">
+                    <div class="sub-sec-left-left">
+                      <img
+                        src="./assets/employees.svg"
+                        alt="Employees Icon"
+                        class="sidebar-employees-icon" />
+                      <h4>Employees</h4>
+                    </div>
+                    <div class="expand-collapsed-icons">
+                      <span class="material-icons-outlined collapsed">
+                        chevron_right
+                      </span>
+                      <span class="material-icons-outlined expand">
+                        expand_more
+                      </span>
+                    </div>
+                  </div>
+                  <div class="sub-sec-right"></div>
+                </div>
+                <div class="employee-sub-lists sub-lists active">
+                  <ul>
+                    <li>All</li>
+                    <li>Active</li>
+                    <li>Inactive</li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="sidebar-lists departments-section"
+            onclick="toggleSideBarListClass(this,['.departments','.sidebar-lists'])">
```

Comment:
where is code of toggleSideBarListClass? if it is not present, please remove it.